File: Chain/Chain/chain/requests/PersonalTrainingRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ZTP_Zadanie3.chain.wnioski
{
    class PersonalTrainingRequest : Request
    {
        public PersonalTrainingRequest(string title, DateTime date)
        {
            this.date = date;
            this.title = title;
        }
    }
}

File: ChainJava/src/chain/requests/PersonalTrainingRequest.java
package chain.requests;

import java.util.Date;

/**
 * Created by Karolina_2 on 2015-04-01.
 */
public class PersonalTrainingRequest extends Request {

    public PersonalTrainingRequest(String nazwa, Date date){
        this.date = date;
        this.nazwa = nazwa;
    }
}
